util/redisutil/redis_coordinator.go

```diff
@@ -40,7 +38,7 @@ func NewRedisCoordinator(redisUrl string) (*RedisCoordinator, error) {
 	}, nil
 }
 
-// This sequencer is live and top priority
+// RecommendLiveSequencer checks this sequencer is live and top priority
```

Comment:
Hmm, this commentary doesn't seem to match the implementation. A check seems like a form of validation, but this is a getter function. Could this be improved?

Reply:
Yeah I'd say "returns the top priority live sequencer" instead of the current comment